### PR TITLE
Supervisor view, create, edit, destroy task

### PR DESCRIPTION
**Pull request by nguyenquanganhc**
## Related Tickets
- #40454

## WHAT (optional)
- Supervisor can view, create, edit, destroy task

## HOW
- Create TaskController, TaskView

## Evidence (Screenshot or Video)
image
image

### CODE REVIEW

**Comment by ManhLeDuc** [comment]
OK

**Comment by trannguyen61** [comment]
OK

**Comment by nguyenquanganhc** [comment]
Ready

File: app/views/courses/_user_item.html.erb [hunk 1]
```diff
@@ -3,7 +3,7 @@
   <td><%= user_item.user_name %></td>
   <td><%= user_item.user_email %></td>
   <td>
-    <% if current_user != user_item.user %>
+    <% if current_user != user_item.user%>
```
Comment (ngocvt-0484) [inline app/views/courses/_user_item.html.erb]:
```suggestion
    <% if current_user != user_item.user %>
```

File: app/views/subjects/show.html.erb [hunk 2]
```diff
@@ -0,0 +1,22 @@
+
```
Comment (ngocvt-0484) [inline app/views/subjects/show.html.erb]: thừa line

**Comment by nguyenquanganhc** [comment]
Ready

**Pull request merged**